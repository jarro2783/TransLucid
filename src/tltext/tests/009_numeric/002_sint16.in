%%
sint16"0";;
sint16"1";;
sint16"-1";;
sint16"32767";;
sint16"-32768";;

sint16"-100" + sint16"200";;

sint16"-100" - sint16"-200";;

sint16"5" * sint16"-3";;

sint16"25" % sint16"7";;

sint16"24" / sint16"7";;
