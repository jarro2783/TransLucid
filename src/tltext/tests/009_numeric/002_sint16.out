sint16"0"
sint16"1"
sint16"12"
sint16"32767"
sint16"-32768"
sint16"0"
sint16"15"
sint16"15"
sint16"13"
sint16"2"
sint16"-123"
sint16"0"
sint16"10"
sint16"-10"
sint16"12"
sint16"6"
sint16"0"
sint16"-20"
sint16"20"
sint16"5"
sint16"4"
sint16"-5"
sint16"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
16
true
